shared/app.js

```diff
@@ -73,7 +73,7 @@ module.exports = Backbone.Model.extend({
      * Initialize the `ClientRouter` on the client-side.
      */
     if (!isServer) {
-      var ClientRouter = this.options.ClientRouter || require(defaultRouterModule);
+      var ClientRouter = this.options.ClientRouter || this.modelUtils._requireAMD(defaultRouterModule);
```

Comment:
hah yer too quick, yeah, i did it really quickly and went "crap, this won't work..." i renamed this to be a work in progress and added a checkbox for the task to make this a callback. i'd be realllyyy surprised if this worked at all before.  This will probably require more work to get things to work, and i need to setup a requirejs env to really test.